R/exploristics_colour.R

```diff
@@ -73,29 +72,29 @@ exploristics_colour <-
     } else {
       if (isTRUE(rev_gradient)) {
         # continuous
-        if (colourPalette == "Expl_Blue") {
-          plot <- plot + scale_color_gradientn(colours = rev(ExploristicsTheme::Expl_Blue))
+        if (colourPalette == "Expl_RGB") {
+          plot <- plot + scale_color_gradientn(colours = rev(ExploristicsTheme::Expl_RGB[c(1, 4, 2)]))
         }
-        if (colourPalette == "Expl_External") {
+        if (colourPalette == "Expl_RGB_dark") {
           plot <-
-            plot + scale_color_gradientn(colours = rev(ExploristicsTheme::Expl_External[c(1, 6, 2)]))
+            plot + scale_color_gradientn(colours = rev(ExploristicsTheme::Expl_RGB_dark[c(1, 4, 2)]))
         }
-        if (colourPalette == "Expl_HighCont") {
+        if (colourPalette == "Expl_Mono") {
           plot <-
-            plot + scale_color_gradientn(colours = rev(ExploristicsTheme::Expl_HighCont[c(1, 2, 4)]))
+            plot + scale_color_gradientn(colours = rev(ExploristicsTheme::Expl_Mono[c(5, 3, 2)]))
         }
       } else {
         # continuous
-        if (colourPalette == "Expl_Blue") {
-          plot <- plot + scale_color_gradientn(colours = ExploristicsTheme::Expl_Blue)
+        if (colourPalette == "Expl_RGB") {
+          plot <- plot + scale_color_gradientn(colours = ExploristicsTheme::Expl_RGB[c(1, 4, 2)])
         }
-        if (colourPalette == "Expl_External") {
+        if (colourPalette == "Expl_RGB_dark") {
           plot <-
-            plot + scale_color_gradientn(colours = ExploristicsTheme::Expl_External[c(1, 6, 2)])
+            plot + scale_color_gradientn(colours = ExploristicsTheme::Expl_RGB_dark[c(1, 4, 2)])
         }
-        if (colourPalette == "Expl_HighCont") {
+        if (colourPalette == "Expl_Mono") {
           plot <-
```

Comment:
Nested if else statements will flag up a warning in the code quality - it's known as cyclomatic complexity. 

I'm happy to leave it in - if you've not used it before the `switch` statement in R is similar to `case_when` from dplyr. It makes code easier to follow and maintain.